bliss/encoder.py

```diff
@@ -32,7 +34,9 @@ def __init__(
         self,
         detection_encoder: DetectionEncoder,
         binary_encoder: Optional[BinaryEncoder] = None,
-        galaxy_encoder: Optional[GalaxyEncoder] = None,
+        galaxy_encoder: Optional[Union[GalaxyEncoder, GalsimEncoder]] = None,
+        lensing_binary_encoder: Optional[BinaryEncoder] = None,
```

Comment:
Let's make LensingBinaryEncoder a new module. Better to repeat ourselves a little now than to entangle these two separate parts of BLISS